app/visualization.py

```diff
@@ -0,0 +1,67 @@
+from app.requester import ExchangeRateRequester
```

Comment:
Application-specific imports should be at the bottom 
https://stackoverflow.com/a/42701457